pkg/backends/datadog/datadog.go

```diff
@@ -56,11 +56,11 @@ var (
 
 // Client represents a Datadog client.
 type Client struct {
-	batchesCreated uint64 // Accumulated number of batches created
-	batchesRetried uint64 // Accumulated number of batches retried (first send is not a retry)
-	batchesDropped uint64 // Accumulated number of batches aborted (data loss)
-	batchesSent    uint64 // Accumulated number of batches successfully sent
-	seriesSent     uint64 // Accumulated number of series successfully sent
+	batchesCreated uint64            // Accumulated number of batches created
+	batchesDropped uint64            // Accumulated number of batches aborted (data loss)
+	batchesSent    uint64            // Accumulated number of batches successfully sent
+	seriesSent     uint64            // Accumulated number of series successfully sent
+	batchesRetried stats.ChangeGauge // Accumulated number of batches retried (first send is not a retry)
```

Comment:
I re-ordered these to make it explicit that the uint64s are at the top (required for atomic alignment).  Technically not required because stats.ChangeGauge is all uint64s as well, but I don't want to bank on that.